geniza/corpus/tests/test_corpus_views.py

```diff
@@ -255,6 +315,76 @@ def test_get_context_data(self, rf):
             == docsearch_view.queryset.get_highlighting.return_value
         )
 
+    def test_scholarship_sort(self, empty_solr, source):
+        """integration test for sorting by scholarship asc and desc"""
+
+        doc_no_record = Document.objects.create()
```

Comment:
you could use the existing `document` and `join` fixtures here; I don't feel strongly, not sure how much it helps with readabiilty